ethereumkit/src/main/java/org/p2p/ethereumkit/external/repository/EthereumRepository.kt

```diff
@@ -2,11 +2,16 @@ package org.p2p.ethereumkit.external.repository
 
 import java.math.BigInteger
 import org.p2p.core.token.Token
+import org.p2p.core.wrapper.HexString
 import org.p2p.core.wrapper.eth.EthAddress
+import org.p2p.ethereumkit.internal.models.RawTransaction
+import org.p2p.ethereumkit.internal.models.Signature
 
 interface EthereumRepository {
     fun init(seedPhrase: List<String>)
     suspend fun getBalance(): BigInteger
     suspend fun loadWalletTokens(): List<Token.Eth>
     suspend fun getAddress(): EthAddress
+    fun getPrivateKey(): BigInteger
```

Comment:
which private key are you getting here?

Reply:
It's Ethereum Repository, so it's eth private key )